.npmignore

```diff
@@ -0,0 +1,9 @@
+.editorconfig
+.gitattributes
+.gitignore
+.npmignore
+.travis.yml
+bower_components
+node_modules
+sample
```

Comment:
Yeah I was thinking that too.  We hadn't done that before though, so I kept it out.  I'll include it if you're cool with it.